rfc6962-bis.xml

```diff
@@ -1471,17 +1471,14 @@ but it is expected there will be a variety.
               <t>Both identities are DNS names, have the same number of labels, and each label in the subjectAltName identity has a matching label at the same position in the redactedSubjectAltName identity. Two DNS labels are matching if either:
                 <list style="symbols">
                   <t>The two labels are identical; or,</t>
-                  <t>The label from the redactedSubjectAltName identity is equal to REDACT(label from subjectAltName identity) (<xref target="redacting_labels"/>)</t>
+                  <t>Neither label is <spanx style="verb">*</spanx> and the label from the redactedSubjectAltName identity is equal to REDACT(label from subjectAltName identity) (<xref target="redacting_labels"/>).</t>
```

Comment:
When * is used, it MUST be unredacted, and so both labels MUST be *.

The "The two labels are identical" case covers this.